src/audio_sdl_mixer.cpp

```diff
@@ -625,7 +625,7 @@ void SdlMixerAudio::SE_Play(std::string const& file, int volume, int pitch) {
 		Output::Warning("Couldn't play %s SE. %s", FileFinder::GetPathInsideGamePath(file).c_str(), Mix_GetError());
 		return;
 	}
-	sounds[channel] = snd_data;
+	sounds[channel] = std::move(snd_data);
```

Comment:
Is it ever safe to copy this object?

If not we should `delete` the copy operations and `default` the move ones.

Reply:
That struct is specific to SDLMixer Audio. Will change it based on your suggestion.